docker-compose.yml

```diff
@@ -51,6 +51,15 @@ services:
       options:
         max-size: "512m"
         max-file: "4"
+  solr_updater:
+    restart: always
+    hostname: "$HOSTNAME"
+    environment:
+      - OL_CONFIG=/olsystem/etc/openlibrary.yml
```

Comment:
Use the conf/openlibrary.yml by default, and override it with the olsystem one in docker-compose.production.yml